controllers/flowcollector_controller.go

```diff
@@ -155,19 +144,6 @@ func (r *FlowCollectorReconciler) reconcile(ctx context.Context, clh *helper.Cli
 		}
 	}
 
-	// OVS config map for CNO
-	if r.mgr.HasCNO() {
-		ovsConfigController := ovs.NewFlowsConfigCNOController(&reconcilersInfo, desired.Spec.Agent.IPFIX.ClusterNetworkOperator.Namespace, ovsFlowsConfigMapName)
-		if err := ovsConfigController.Reconcile(ctx, desired); err != nil {
-			return r.status.Error("ReconcileCNOFailed", err)
-		}
-	} else {
-		ovsConfigController := ovs.NewFlowsConfigOVNKController(&reconcilersInfo, desired.Spec.Agent.IPFIX.OVNKubernetes)
-		if err := ovsConfigController.Reconcile(ctx, desired); err != nil {
-			return r.status.Error("ReconcileOVNKFailed", err)
-		}
-	}
-
 	// eBPF agent
 	ebpfAgentController := ebpf.NewAgentController(reconcilersInfo.NewInstance(r.mgr.Config.EBPFAgentImage, r.status))
```

Comment:
Why not adding a test on agent type here and crash if IPFIX ?

That would be more relevent in case someone still uses this type

Reply:
I hesitated between the 2 approaches.. the thing I like with switching automatically to eBPF is that we completely remove any legacy IPFIX stuff from the code (except in the API) even that simple bool check & failure, so the state of the art is now: you have no choice it's eBPF. I have the feeling this is more user friendly than triggering an error & avoids risks of misconfiguration.